containers/server-saline-image/cleanup.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+find /etc -maxdepth 1 -type f -name '*-' -delete
```

Comment:
A bit risky

Reply:
It's just intended to delete backup files created on adding salt user and group, maybe it makes sense to remove them explicitly, but there shouldn't be any other files like these ones.